### PR TITLE
Upgrade jquery to 1.6.3 due to CVE-2011-4969

### PR DESCRIPTION
**Pull request by Astralidea**
## What is the purpose of the change

*(For example: This pull request improves file read performance by buffering data, fixing AVRO-XXXX.)*
Our security scanning software has found some security issues, so we want to update the relevant version library. Do we need to raise a JIRA issue?

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Documentation

- Does this pull request introduce a new feature? (no)

### CODE REVIEW

**Review by martin-g — APPROVED** [review]

File: lang/java/ipc/src/main/velocity/org/apache/avro/ipc/stats/templates/statsview.vm [hunk 1]
```diff
@@ -51,7 +51,7 @@
 <title>$title</title>
 <script type="text/javascript" src="static/protovis-r3.2.js"></script>
 <script type="text/javascript" src="static/tipsy.js"></script>
-<script src="static/jquery-1.4.2.min.js" type="text/javascript"></script>
```
Comment (Fokko) [inline lang/java/ipc/src/main/velocity/org/apache/avro/ipc/stats/templates/statsview.vm:54]: Shouldn't we pull these resources from a CDN? https://code.jquery.com/jquery-1.6.3.min.js

Comment (Gerrit0) [reply lang/java/ipc/src/main/velocity/org/apache/avro/ipc/stats/templates/statsview.vm:54]: IMO, no. <https://httptoolkit.com/blog/public-cdn-risks/>

**Review by Fokko — APPROVED** [review]
```
➜  avro-jquery curl https://raw.githubusercontent.com/apache/avro/f2d0fcf8bfafec2cc155dba2c7fd25ec028b6d16/lang/java/ipc/src/main/velocity/org/apache/avro/ipc/stats/static/jquery-1.6.3.min.js > jquery.github.js
curl https://code.jquery.com/jquery-1.6.3.min.js > jquery.cdn.js
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 91626  100 91626    0     0  1361k      0 --:--:-- --:--:-- --:--:-- 1376k
drwxr-xr-x   5 fokko.driesprong  wheel   160B Jul 15 09:22 .
drwxrwxrwt  15 root              wheel   480B Jul 15 09:21 ..
-rw-r--r--   1 fokko.driesprong  wheel    89K Jul 15 09:22 jquery.cdn.js
-rw-r--r--   1 fokko.driesprong  wheel    89K Jul 15 09:21 jquery.github.js
➜  avro-jquery diff jquery.cdn.js jquery.github.js 
➜  avro-jquery echo $?
0

**Comment by Fokko** [comment]
Thanks @Gerrit0 for fixing this 👍

**Comment by Gerrit0** [comment]
Wasn't me this time - all credit to @Astralidea

**Comment by Fokko** [comment]
@Gerrit0 Thank you, you popped up first, sorry for that. Thanks @Astralidea 🙌 much appreciated!

**Pull request merged**